### DISCUSSION TITLE
API rules for request origin

### DISCUSSION THREAD

**Discussion by Freethepizza** [pocketbase/pocketbase#1467] [Q&A]
Hi everybody,

Is there any way to filter wich request URL's can list a collection?
I've been trying with @request.data.referer = 'https://example.com' ,  @request.data.url= 'https://example.com'  and more without any success.

Thanks in advance

**Comment by ganigeorgiev** [comment]
No, this is not possible at the moment with the `@request.*` filter.
`@request.data.*` is the body data of your request.
I'll consider in the future to add support for `@request.headers.*`.

But for now, if you want to filter a request based on a header value, you'll have to [use PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) and [register a middleware](https://pocketbase.io/docs/router/#middlewares).

**Reply by ganigeorgiev** [reply]
Alternatively, if you are trying to implement a cors-like behavior, you can start the executable with `--origins`:

```sh
./pocketbase serve --origins='example1.com,example2.com'
```
